src/components/Views/SyncView/index.js

```diff
@@ -17,16 +16,14 @@ import type { StateProps, DispatchProps } from './SyncView';
 
 function mapStateToProps(state: StoreState): StateProps {
   return {
-    devices: values(state.devices),
-    syncTarget: state.selectedDevice
+    devices: values(state.devices)
```

Comment:
there is no 'syncTarget' in the UI, perhaps this was a half-implemented feature that didn't go anywhere yet.